repair/repair.hh

```diff
@@ -498,6 +504,8 @@ struct node_ops_cmd_request {
     utils::UUID ops_uuid;
     std::list<gms::inet_address> ignore_nodes;
     std::list<gms::inet_address> leaving_nodes;
+    // Map existing nodes to replacing nodes
+    std::unordered_map<gms::inet_address, gms::inet_address> replace_nodes;
```

Comment:
Ah, here is the comment. But it should also be present in the IDL.

Reply:
OK.  I will add to IDL too.